font-icons/examples-icons.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Examples Icons
+page_title: .NET MAUI Font Icons - Overview
+description: Telerik font icons are collections of small vector graphics used across the components in the Telerik UI for .NET MAUI suite.
+slug: telerik-examples-icons
+position: 1
+---
+
+<link rel="stylesheet" href="style-examples.css" />
+
+# Telerik UI for .NET MAUI Font Icons
+
+You can choose any of the available Telerik Font icons: 
```

Comment:
In addition to the suggested change, think about expanding the intro a bit. What are Examples Icons, where would people use them or where are they visible in action?

```suggestion
You can choose any of the available Telerik font icons: 
```